wordvecspace/command.py

```diff
@@ -94,17 +107,17 @@ def define_subcommands(self, subcommands):
         convert_cmd.set_defaults(func=self.convert)
         convert_cmd.add_argument('input_dir',
             help='Input directory containing Google Word2Vec format files'
-                 ' (vocab.txt, vectors.bin)')
-        convert_cmd.add_argument('output_file',
-            help='Output file where WordVecSpace format files are produced')
+                                 ' (vocab.txt, vectors.bin)')
```

Comment:
@prasannababuAddagiri Can you please check the indentation here.

Reply:
agree @chatrapathik